admin.iptools.user.js

```diff
@@ -349,19 +349,38 @@ body[noipspls] .tag_key_ip {
      }
   }
 
+  function tickAll() {
+    RoomInstance.extensions.filelist.filelist.forEach(file => {
+      if (!file.dom.fileElement.classList.contains("file_selected")) {
+        file.dom.controlElement.firstChild.click()
```

Comment:
>no semicolon

Reply:
`file.setData("checked", true)` should work betterly actually...
No need to check if it's checked at all.